tutorials/buoyantSimpleFoam-aster/fluid/system/changeDictionary-1

```diff
@@ -0,0 +1,32 @@
+// Re = 500, Pr = 0.01, k = 1
```

Comment:
Do we still need this file? It would only make sense for parametric studies. Do we refer to it?

Reply:
No we don't need it, it runs perfectly without. I have deleted these changeDictionary files.